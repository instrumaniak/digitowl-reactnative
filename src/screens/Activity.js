import React from 'react'

//import { View, FlatList } from 'react-native'

import {
  Container,
  Content,
  Header,
  Body,
  Title
} from 'native-base'

import ActivityCard from '../components/ActivityCard'
import { colors } from '../config'

const Activity = () => (
  <Container style={{ backgroundColor: colors.whiteSmoke }}>
    <Header hasTabs style={{ elevation:2, paddingLeft: 20, backgroundColor: colors.green }} androidStatusBarColor={colors.greenDark}>
      <Body><Title>Activity</Title></Body>
    </Header>
    <Content removeClippedSubviews={true} contentContainerStyle={{paddingHorizontal: 10, paddingBottom: 10, paddingTop: 5}}>
      <ActivityCard />
      <ActivityCard />
      <ActivityCard />
    </Content>
  </Container>
)

// const Activity = () => (
//   <View style={{ backgroundColor: colors.whiteSmoke, flex: 1 }}>
//     <Header hasTabs style={{ elevation:2, paddingLeft: 20, backgroundColor: colors.green }} androidStatusBarColor={colors.greenDark}>
//       <Body><Title>Activity</Title></Body>
//     </Header>
//     <View style={{marginBottom: 40, paddingHorizontal: 10, paddingBottom: 10, paddingTop: 5}}>
//       <FlatList
//         data={new Array(10).fill().map((x,i) => i)}
//         keyExtractor={(item, index) => `${index}`}
//         renderItem={() => <ActivityCard />}
//       />
//     </View>
//   </View>
// )

export default Activity
