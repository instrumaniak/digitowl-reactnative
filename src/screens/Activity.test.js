/**
 *   Tests for component: Activity
 */

import React from 'react'
import Activity from './Activity'
import renderer from 'react-test-renderer'

test('Renders correctly', () => {
  const tree = renderer.create(<Activity />).toJSON()
  expect(tree).toMatchSnapshot()
})
