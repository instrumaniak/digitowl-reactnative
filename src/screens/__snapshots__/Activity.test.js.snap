// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "#F5F3F5",
      },
    ]
  }
>
  <View
    onLayout={[Function]}
  >
    <View
      androidStatusBarColor="#4A687F"
      hasTabs={true}
      style={
        Array [
          Object {
            "backgroundColor": "#F8F8F8",
            "borderBottomColor": "#a7a6ab",
            "borderBottomWidth": null,
            "elevation": 0,
            "flexDirection": "row",
            "height": 64,
            "justifyContent": "center",
            "left": 0,
            "paddingLeft": 6,
            "paddingRight": 10,
            "paddingTop": 18,
            "right": 0,
            "shadowColor": null,
            "shadowOffset": null,
            "shadowOpacity": null,
            "shadowRadius": null,
            "top": 0,
          },
          Object {
            "backgroundColor": "#4E6D86",
            "elevation": 2,
            "paddingLeft": 20,
          },
        ]
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "flex": 1,
          }
        }
      >
        <Text
          numberOfLines={1}
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 17,
                "fontWeight": "700",
                "marginLeft": undefined,
                "paddingLeft": 4,
                "paddingTop": 1,
                "textAlign": "center",
              },
              Object {
                "backgroundColor": "transparent",
                "color": "#FFFFFF",
                "fontFamily": "System",
                "fontSize": 20,
                "fontWeight": "400",
                "letterSpacing": 0.361328,
                "lineHeight": 25,
              },
            ]
          }
        >
          Activity
        </Text>
      </View>
    </View>
  </View>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        Object {
          "paddingBottom": 10,
          "paddingHorizontal": 10,
          "paddingTop": 5,
        },
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    removeClippedSubviews={true}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderRadius": 2,
            "elevation": 2,
            "flexDirection": "row",
            "marginBottom": 5,
            "padding": 10,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 15,
              "paddingLeft": 2,
            }
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#5B6267",
                "fontFamily": "System",
                "fontSize": 17,
                "fontWeight": "400",
                "letterSpacing": -0.408,
                "lineHeight": 22,
                "marginBottom": 10,
              }
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam dictum ante vitae tortor vulputate, at posuere ante lacinia.
          </Text>
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#c3c6c7",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
                "letterSpacing": -0.24,
                "lineHeight": 20,
              }
            }
          >
            3 min. ago, 1:45 PM
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#c3c6c7",
                  },
                ],
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderRadius": 2,
            "elevation": 2,
            "flexDirection": "row",
            "marginBottom": 5,
            "padding": 10,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 15,
              "paddingLeft": 2,
            }
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#5B6267",
                "fontFamily": "System",
                "fontSize": 17,
                "fontWeight": "400",
                "letterSpacing": -0.408,
                "lineHeight": 22,
                "marginBottom": 10,
              }
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam dictum ante vitae tortor vulputate, at posuere ante lacinia.
          </Text>
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#c3c6c7",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
                "letterSpacing": -0.24,
                "lineHeight": 20,
              }
            }
          >
            3 min. ago, 1:45 PM
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#c3c6c7",
                  },
                ],
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderRadius": 2,
            "elevation": 2,
            "flexDirection": "row",
            "marginBottom": 5,
            "padding": 10,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 15,
              "paddingLeft": 2,
            }
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#5B6267",
                "fontFamily": "System",
                "fontSize": 17,
                "fontWeight": "400",
                "letterSpacing": -0.408,
                "lineHeight": 22,
                "marginBottom": 10,
              }
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam dictum ante vitae tortor vulputate, at posuere ante lacinia.
          </Text>
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#c3c6c7",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
                "letterSpacing": -0.24,
                "lineHeight": 20,
              }
            }
          >
            3 min. ago, 1:45 PM
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#c3c6c7",
                  },
                ],
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
